.github/workflows/pytest-gpu.yml

```diff
@@ -78,7 +78,10 @@ jobs:
 
     - name: Build docker image
       run: |
-          docker build . --pull --file docker/Dockerfile.devito --tag devito_img --build-arg base=${{ matrix.base }}
+          docker build . --rm --pull --file docker/Dockerfile.devito --tag devito_img --build-arg base=${{ matrix.base }}
+
+    - name: Remove dangling layers
```

Comment:
So why is it needed again?

Reply:
~300 GB of layers on v1003 and counting, which I purged manully for now

if we pushed too much in a good week before docker-build kicks in (which does purge IIRC), we might end up saturating the hard drive

though now that I think about it, that's weird, the base image shouldn't get rebuilt each time... mh

Reply:
If there is ~300 GB of layers then there is something else going on probably due to
- cancelled workflow
- Dev on the machine

So this may "fix" some of it but it should be safer to have the actual machine have a safety cron job with that